packages/app/cypress/e2e/index.cy.ts

```diff
@@ -1,5 +1,19 @@
 import defaultMessages from '@packages/frontend-shared/src/locales/en-US.json'
 
+function getPathForPlatform (posixPath: string) {
```

Comment:
Could we import this?

```
import { getPathForPlatform } from '../../src/paths'
```

Reply:
I purposely duplicated this logic so a regression in `src/paths` wouldn't cause the tests to erroneously pass by virtue of changing the expectation of the tests. I figure it's worth the duplication it to ensure the expectations of this spec are kept constant.

Reply:
this is `path.normalize()`...